### PR TITLE
#Homework Pizza Mocks

### PR DESCRIPTION
**Pull request by Vicx95**

### CODE REVIEW

**Review by MaciejSalamonski — APPROVED** [review]
I don't have any comments. Nice code ;~}

File: test/PizzeriaTest.cpp [hunk 1]
```diff
@@ -65,3 +64,36 @@ TEST_F(PizzeriaTest, calculatePriceForPizzaMock)
...
+    NiceMock<PizzaMock> pm2;
+    Pizzas pizzas = {new PizzaStub{"stub"}, &pm1, &pm2};
+
+    EXPECT_CALL(pm1, getPrice()).WillOnce(Return(mockFirstPrice));
+    EXPECT_CALL(pm1, getName()).WillOnce(Return(mockFirstName));
+    EXPECT_CALL(pm1, getBakingTime()).WillOnce(Return(mockFirstBakingTime));
+    EXPECT_CALL(pm2, getPrice()).WillOnce(Return(mockSecondPrice));
+    EXPECT_CALL(pm2, getName()).WillOnce(Return(mockSecondName));
+    EXPECT_CALL(pm2, getBakingTime()).WillOnce(Return(mockSecondBakingTime));
+    EXPECT_CALL(tm, sleep_for).Times(3);
```
Comment (ziobron) [inline test/PizzeriaTest.cpp:89]: Not this way. You should expect that sleep_for is called with a proper duration - the baking time of each pizza.

**Review by ziobron — CHANGES_REQUESTED** [review]
One thing to correct.

**Review by WBigaj — COMMENTED** [review]
I have checked your code and have nothing to add.

File: test/PizzeriaTest.cpp (hunk 1 above)
Comment (Vicx95) [reply test/PizzeriaTest.cpp:89]: @ziobron could you explain? I think that is good solution

**Review by ziobron — APPROVED** [review]

**Comment by ziobron** [comment]
{ "requirement_ids" : [1119, 1119, 1119, 3524, 3525, 3526] }
3 * 3 XP za punktualność (1119)
10 XP za TimeMock w Pizzas (3524)
15 XP za nowy przypadek testowy w Pizzas (3525)
5 XP za workflow w Pizzas (3526)

**Pull request closed without merging**